Formula/kvrocks.rb

```diff
@@ -0,0 +1,38 @@
+class Kvrocks < Formula
+  desc "Key-value NoSQL database based on RocksDB and compatible with Redis protocol"
+  homepage "https://github.com/bitleak/kvrocks/"
+  url "https://github.com/bitleak/kvrocks.git",
+    tag:      "v2.0.1",
+    revision: "5f41145adb25a79706a68d233a28a0f633ef255f"
+  license "BSD-3-Clause"
+  head "https://github.com/bitleak/kvrocks.git", branch: "unstable"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "git", "submodule", "init"
+    system "git", "submodule", "update"
+    mkdir "build" do
+      system "cmake", "-DCMAKE_BUILD_TYPE=Release", "..", *std_cmake_args
+      system "make", "-j4", "kvrocks"
```

Comment:
```suggestion
      system "make", "kvrocks"
```
`brew` sets the number of jobs correctly, so there shouldn't be a need to hard-code a number that could be wrong. (It's also wrong for our CI nodes.)